cypress/e2e/po/pages/account-api-keys.po.ts

```diff
@@ -60,6 +60,10 @@ export default class AccountPagePo extends PagePo {
     return this.self().get('button[type="reset"]').click();
   }
 
+  changePasswordModal() {
+    return cy.get('.change-password-modal');
```

Comment:
we should use `data-testid` in order to be specific in regards to our targets. Classes can change without us realising that they are bound to a test